.github/workflows/pythonpackage-windows.yml

```diff
@@ -0,0 +1,49 @@
+name: Python package
+
+on:
+  pull_request:
+    branches:
+    - master
+  push:
+    branches:
+    - master
+
+jobs:
+  build:
+    name: Test on python ${{ matrix.python-version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 3
+      matrix:
+        python-version: [2.7, 3.6, 3.7]
+        os: [windows-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install urllib3 certifi pytz pyflakes faker nose wget
+    - name: Test with nosetests
+      run: |
+        $ErrorActionPreference = 'continue'
+        pyflakes minio/*.py
+        python setup.py install
+        python setup.py nosetests
+    - name: Test with functional tests
+      env:
+        MINT_MODE: full
+        SERVER_ENDPOINT: localhost:9000
+        ACCESS_KEY: minio
+        SECRET_KEY: minio123
+        ENABLE_HTTPS: 0
```

Comment:
It is better to keep a cert and enable HTTPS for encryption-related tests @vadmeste just like ubuntu-latest

Reply:
> It is better to keep a cert and enable HTTPS for encryption-related tests @vadmeste just like ubuntu-latest

It is little hard to generate certificates in Windows, it seems it is easier to create a certificate and commit in minio-py, like what's done in minio-go

Reply:
Commit to the minio-py don't autogenerate it @vadmeste